test/e2e/testing-manifests/ingress/static-ip-2/rc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ReplicationController
```

Comment:
Should we upgrade this to deployment?

Reply:
We can also change the rc to a deployment for static-ip/rc.yaml. I'll make that change in a separate PR.

Reply:
See issue #58629